invenio_communities/models.py

```diff
@@ -311,11 +316,19 @@ def remove_record(self, record):
         :type record: `invenio_records.api.Record`
         """
         key = current_app.config['COMMUNITIES_RECORD_KEY']
+        access_key = '_access'
 
         assert self.id in record.get(key, [])
-
         record[key] = [c for c in record[key] if c != self.id]
 
+        assert self.id in record.get(access_key, {'read': []})['read']
```

Comment:
Again, not sure of the assert... Do we want it to crash if the value is not in the record?